cache/in-memory/src/config.rs

```diff
@@ -69,72 +41,59 @@ impl Config {
     pub fn message_cache_size_mut(&mut self) -> &mut usize {
         &mut self.message_cache_size
     }
+    /// Returns an immutable reference to the event types enabled.
```

Comment:
change the docs here

Reply:
I never saved that file doing that change apparently. Updated